django/contrib/auth/forms.py

```diff
@@ -163,7 +163,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         password = self.fields.get("password")
         if password:
-            password.help_text = password.help_text.format("../password/")
+            password.help_text = password.help_text.format(
+                f"../../{self.instance.pk}/password/"
```

Comment:
The f-string here should be dropped too, correct?

Reply:
No, here the f-string contains only property access, so it's fine.